src/windows/Accessibility.ts

```diff
@@ -3,17 +3,35 @@
 *
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT license.
-*
-* Web wrapper for subscribing or querying the current state of the
-* screen reader.
 */
 
-import { Accessibility as CommonAccessibility } from '../common/Accessibility';
+import RN = require('react-native');
+import SubscribableEvent from 'subscribableevent';
+import { Accessibility as NativeAccessibility } from '../native-common/Accessibility';
+import SyncTasks = require('synctasks');
+
+export class Accessibility extends NativeAccessibility {
+    private _isHighContrast = RN.AccessibilityInfo && RN.AccessibilityInfo.initialHighContrast;
```

Comment:
>RN.AccessibilityInfo [](start = 30, length = 20)

This is a Windows-only file so let's assume RN.AccessibilityInfo exists and get rid of this.

Reply:
Fixed